parsers/plmdcaparser.py

```diff
@@ -7,13 +7,17 @@ def PlmdcaParser(input):
     output = []
 
     for line in contents:
-        line = line.lstrip().split(',')
+        line = line.replace(",", " ")
+        line = line.lstrip().split(' ')
```

Comment:
You first replace commas with white spaces and then split by white spaces. Why not spiting by commas on the first place? If rear white spaces where a problem in previous version, maybe add `rstrip()`. Replace those two lines with `line=line.lstrip().rstrip().split(',')`